apps/rabbitci_core/web/controllers/build_controller.ex

```diff
@@ -74,4 +103,7 @@ defmodule RabbitCICore.BuildController do
         |> render("show.json")
     end
   end
+  def show(conn, %{"build_number" => bn}) when not is_integer(bn) do
```

Comment:
unless you pre-preprocessed the params elsewhere, `bn` will always be a string (all param keys and values are strings)